daemon/daemon.go

```diff
@@ -154,11 +154,11 @@ func (daemon *Daemon) containerRoot(id string) string {
 // Load reads the contents of a container from disk
 // This is typically done at startup.
 func (daemon *Daemon) load(id string) (*Container, error) {
-	container := &Container{
-		root:         daemon.containerRoot(id),
-		State:        NewState(),
-		execCommands: newExecStore(),
-	}
+	container := &Container{}
+	container.root = daemon.containerRoot(id)
+	container.State = NewState()
+	container.execCommands = newExecStore()
```

Comment:
Ahhhh, lightbulb moment. That makes sense. I just learnt more about go. Will verify and address in the morning, Thanks!